File: src/main/java/com/ploy/bubble_server_v3/common/jwt/JwtProperties.java
package com.ploy.bubble_server_v3.common.jwt;


import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Getter
@Setter
@Component
@ConfigurationProperties(prefix = "jwt")
public class JwtProperties {
    private String issuer;
    private String clientSecret;
    private Long tokenExpire;
    private Long refreshTokenExpire;
}

File: src/main/kotlin/com/ploy/bubble_server_v2/common/jwt/JwtProperties.kt
package com.ploy.bubble_server_v2.common.jwt

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

@Component
@ConfigurationProperties(prefix = "jwt")
data class JwtProperties(
    var issuer: String? = null,
    var clientSecret: String? = null,
    var tokenExpire: Int = 0,
    var refreshTokenExpire: Int = 0
)